workflow/usage.rst

```diff
@@ -175,12 +175,63 @@ what actions are allowed on a blog post::
     $transitions = $workflow->getEnabledTransitions($post);
 
 Using Events
-------------
+============
 
 To make your workflows even more powerful you could construct the ``Workflow``
-object with an ``EventDispatcher``. You can now create event listeners to
-block transitions (i.e. depending on the data in the blog post). The following
-events are dispatched:
+object with an ``EventDispatcher``. If you are using the ``Workflow`` component within
+the Symfony standard edition, this is already done for you.
+
+When a workflow subject's state is updated, the component dispatches several events that
+enable you to alter the component's behaviour:
+
+* ``workflow.[workflow name].guard.[transition name]``: dispatched before applying a transition.
```

Comment:
Actually, I'm not sure what's the point of the generic `workflow.guard`/`leave`/`transition` etc. events. Logging of each and every update for all available workflows?
Same for events having a `[workflow name]` (e.g. `workflow.blogpost.guard`), the only use case I can see here is logging. Guarding every step in a workflow can only be useful for cases when the workflow can be started for a subject that's in an unacceptable state.
Hopefully, we can clarify this a bit?